### PR TITLE
More logs around Insurely flow

### PR DESCRIPTION
**Pull request by alebedev**
## Describe your changes

More debug logs for Insurely to help with debugging

## Checklist before requesting a review

- [ ] I have performed a self-review of my code

### CODE REVIEW

**Comment by alebedev** [comment]
Current dependencies on/for this PR:
* main
  * **PR #3709** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/3709?utm_source=stack-comment-icon" target="_blank">Graphite</a>  👈

This [stack of pull requests](https://stacking.dev/?utm_source=stack-comment) is managed by [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/3709?utm_source=stack-comment).

File: apps/store/src/components/PriceCalculator/FetchInsurance.tsx [hunk 1]
```diff
@@ -110,6 +113,9 @@ export const FetchInsurance = (props: Props) => {
 
   const handleInsurelyCompleted: OnCompleted = useCallback(() => {
     if (dataCollectionId) {
+      LOGGER.info(
+        `Saving Insurely data collection ID: ${dataCollectionId}, priceIntenId: ${props.priceIntentId}`,
```
Comment (gustaveen) [inline apps/store/src/components/PriceCalculator/FetchInsurance.tsx:117]:
```suggestion
        `Saving Insurely data collection ID: ${dataCollectionId}, priceIntentId: ${props.priceIntentId}`,
```

**Review by gustaveen — APPROVED** [review]
👌 Just a small typo

**Comment by alebedev** [comment]
### Merge activity

* **Dec 8, 9:32 AM**: @@alebedev merged this pull request with [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/3709).

**Pull request merged**